source/common/ssl/utility.cc

```diff
@@ -8,10 +8,20 @@
 namespace Envoy {
 namespace Ssl {
 
+const ASN1_TIME& epochASN1_Time() {
+  static ASN1_TIME* e = []() -> ASN1_TIME* {
+    ASN1_TIME* epoch = ASN1_TIME_new();
+    const time_t epoch_time = 0;
+    ASSERT(ASN1_TIME_set(epoch, epoch_time) != NULL);
```

Comment:
This won't be evaluated in release build, so it has to be RELEASE_ASSERT.
Can you merge master? clang_tidy will catch stuff like this.

Reply:
Yeah. I changed it and split it in to two lines with ASSERTing return value